### PR TITLE
[RHO-4346] Log Channel Refactor - Add ability to specify channel for exception logs

### PR DESCRIPTION
**Pull request by jpersons-series**
#### Description

Add ability to specify channel for exception logs.
Pulled out this step from this PR - https://github.com/series-ai/logger-lib/pull/2 - so we can do deprecation after all the refactoring is done to avoid breaking things or generate hundreds of warnings at once.

#### Testing plan

N/A - just sanity with building/running

#### JIRA + Optional Slack context

https://series-ai.atlassian.net/browse/RHO-4346
https://series-ai.atlassian.net/wiki/spaces/Engineering/pages/62980102/Cadmus+Log+Refactor

### CODE REVIEW

**Comment by jpersons-series** [comment]
PR for DLL: https://github.com/series-ai/core-lib/pull/12

**Review by pichangetheworld — APPROVED** [review]

**Comment by jpersons-series** [comment]
Also need accompanying approval so I can merge together: https://github.com/series-ai/core-lib/pull/12

File: PadoruLogger/Debug.cs [hunk 1]
```diff
@@ -160,6 +161,28 @@ public static void LogError(object message, string channel, object context)
...
+        public static void LogException(Exception e, string channel, object context)
+        {
+            LogException(null, channel, e, context);
+        }
+        
+        public static void LogException(object messageHeader, string channel, Exception e, object context)
+        {
+            var message = messageHeader != null ? $"{messageHeader}. {e.Message}" : e.Message;
+            
+            InternalLog(LogType.Exception, message, DEFAULT_CHANNEL_NAME, context, e.StackTrace);
```
Comment (npring) [inline PadoruLogger/Debug.cs]: DEFAULT_CHANNEL_NAME -> channel

**Review by npring — APPROVED** [review]
minor issue, looks good!

File: PadoruLogger/Debug.cs (hunk 1 above)
Comment (jpersons-series) [reply PadoruLogger/Debug.cs]: ahh good catch thanks

**Pull request merged**